### PR TITLE
Fix deploy

### PR DESCRIPTION
**Pull request by samuelgarcia**

### CODE REVIEW

**Review by alejoe91 — APPROVED** [review]

File: .github/workflows/publish-to-pypi.yml [hunk 1]
```diff
@@ -22,6 +22,7 @@ jobs:
           pip install pytest
           pip install zarr
           pip install setuptools wheel twine build
+          pip install -e .[test_core]
           pip install -e .
```
Comment (h-mayorquin) [inline .github/workflows/publish-to-pypi.yml:26]: Isn't this redundant after btw?

Comment (alejoe91) [reply .github/workflows/publish-to-pypi.yml:26]: no, otherwise it doesn't install psutils

Comment (alejoe91) [reply .github/workflows/publish-to-pypi.yml:26]: (that was the first failure)

Comment (h-mayorquin) [reply .github/workflows/publish-to-pypi.yml:26]: I meant the second one. Line 26.

Comment (alejoe91) [reply .github/workflows/publish-to-pypi.yml:26]: sorry. right!

**Pull request merged**